qiita_db/util.py

```diff
@@ -106,6 +107,8 @@ def typecast_string(string):
     The function first tries to convert to an int. If that fails, it tries to
     convert to a float. If that fails it returns the original string.
     """
+    if isinstance(string, datetime):
```

Comment:
This will silently fail and return `None` if the object can't be converted though, right?